app/views/entities/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="w-full relative">
+  <div class="text-center bg-blue top-0 left-0 right-0 py-1 fixed">
+    <h1 class="font-bold text-4xl text-white">Entities <%= @group.entities.size %></h1>
+  </div>
+
+  <div id="entities" class="fixed left-4 top-3">
+    <%= link_to image_tag("arrow.png", class:"max-h-6"), user_groups_path(current_user) %>
+  </div>
+
+  <div id="entities" class="pt-16 pb-12 mx-4 mb-20 bg-light_grey">
+    <%= render @group.entities %>
+  </div>
+
+  <div class="w-full flex flex-row text-center justify-center items-center bg-green py-2 fixed bottom-0 left-0 right-0">
+    <%= link_to "New Transaction", new_user_group_entity_path(current_user, @group), 
+    class: "py-3 px-5 bg-blue-600 text-white block font-medium border py-4 px-20" %>
+  </div>
```

Comment:
- [ ] On the transaction page, the requirement is to have the sum of the total number of transaction of each category at the top of this transaction page. Kindly add it up 👏 

### Requirement
![image](https://github.com/IsmaelMastronardi/budget-app/assets/50990925/221ac23e-ec5b-4b3d-8879-35b8f6733a65)

### View:
![image](https://github.com/IsmaelMastronardi/budget-app/assets/50990925/cf8b7f54-35ad-4f1d-ac7a-f490e817b5d2)